app/api/verify_email/route.ts

```diff
@@ -0,0 +1,33 @@
+import {
```

Comment:
On the verify email route we could probably create a GET call which takes the token ID and redirects the user to a success or an error page depending on what they do (that's the normal pattern)

So have a route that might be `/verify-email?token=eyJhbGciOiJIUzI1NiIsInR`

Reply:
Done ✅